### ISSUE TITLE
Requesting English README.md and documentation

### ISSUE THREAD

**Issue by muntasimfuad** [deltazefiro/Amarok-Hider#12]
Would you mind translating your github page (README.md) & documentation to English so that more people can contribute to your project?  It'll also help many users.

**Comment by deltazefiro** [comment]
Certainly! But that may take some time.

**Comment by deltazefiro** [comment]
By the way, I'm not exactly good at English.  
If anyone who interested in translating the documents, please feel free to make PRs — I would appreciate it.

**Comment by deltazefiro** [comment]
Hey @muntasimfuad !  :clap: 
I finally got the [readme](https://github.com/deltazefiro/Amarok-Hider/blob/main/README.en.md) and the [documentation](https://deltazefiro.github.io/Amarok-doc/en-US/) translated!  
And btw, I came across [your post on reddit](https://www.reddit.com/r/fossdroid/comments/zyclmz/amarok_a_lightweight_app_which_can_hide_private/) when searching for Amarok's fdroid website recently :rofl: Thanks for the post and promoting!

**Comment by muntasimfuad** [comment]
Wow!  You saw my post.

I frequently find  people searching for FOSS private  file hider app like Amarok on different FOSS & android related subreddit. When I first discovered Amarok in IzzyOnDroid & found that it supports Shizuku, I instslled the app & liked it so much, I instantly thought about posting on reddit. I have no problem with Chinese readme & documention. But thinking I am the  dev😅,the mod of the subreddit suggested me to provide English translation. Then clarifying that I not dev, I created this issue.

As translation of both readme & documentation is complete, this issue can be closed.
